2-mandatory/3-financial-times.js

```diff
@@ -14,7 +20,9 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  return allArticleTitles.reduce(function (shortestTitle, word) {
+    return shortestTitle.length <= word.length ? shortestTitle : word;
+  });
```

Comment:
@lalehkoupa  well done for your solution, it looks like you might have some programming background ;)

Reply:
@ellietms my coding background is for long time a go and I forgot everything::),I searched in the google and find that code but the I couldn't understand the concept, asked my Edu buddy(Simon) and explained to me.

Reply:
@lalehkoupa well done, it is great to find solutions in the internet but it would be great to write your own solutions for your home works for improving your technical skills :)

Reply:
@ellietms yes sure Ellie, at first I completed my function with simple for loop and comparison and it did work ,actually I was just curious about using array methods in my function thats why I searched about that to improve my code :)